pkg/machine/ignition/ready.go

```diff
@@ -0,0 +1,36 @@
+package ignition
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/containers/podman/v4/pkg/machine/define"
+)
+
+type ReadyUnitOpts struct {
+	Port uint64
+}
+
+func CreateReadyUnitFile(provider define.VMType, opts *ReadyUnitOpts) (string, error) {
+	readyUnit := DefaultReadyUnitFile()
```

Comment:
Can `DefaultReadyUnitFile()` be integrated into this function?

Reply:
i think so, but it would be my preference to wait on that move as it is bigger